adapters/http-base/src/main/java/org/eclipse/hono/adapter/http/impl/VertxBasedHttpProtocolAdapter.java

```diff
@@ -49,6 +50,10 @@
  */
 public final class VertxBasedHttpProtocolAdapter extends AbstractVertxBasedHttpProtocolAdapter<HttpProtocolAdapterProperties> {
```

Comment:
In the `VertxBasedHttpProtocolAdapter` methods, a body handler now only gets added if `getConfig().isAuthenticationRequired()` is `true`.
That means, the body handler is missing if authentication _isn't_ required/enabled.